src/Workleap.Extensions.Mongo.Tests/MongoIndexerTests.cs

```diff
@@ -15,20 +15,22 @@ public MongoIndexerTests(ConfigurationMongoFixture fixture, ITestOutputHelper te
     [Fact]
     public async Task Indexes_Are_Automatically_Created_When_Specifying_DocumentType()
     {
-        await this.Services.GetRequiredService<IMongoIndexer>().UpdateIndexesAsync(new[] { typeof(PersonDocument) });
-        await this.AssertPersonDocumentIndexes();
+        await this.Services.GetRequiredService<IMongoIndexer>().UpdateIndexesAsync(new[] { typeof(PersonDocument), typeof(OtherDatabasePersonDocument) });
+        await this.AssertPersonDocumentIndexes<PersonDocument>();
+        await this.AssertPersonDocumentIndexes<OtherDatabasePersonDocument>();
     }
 
     [Fact]
     public async Task Indexes_Are_Automatically_Created_When_Specifying_Assembly()
     {
         await this.Services.GetRequiredService<IMongoIndexer>().UpdateIndexesAsync(typeof(PersonDocument).Assembly);
-        await this.AssertPersonDocumentIndexes();
+        await this.AssertPersonDocumentIndexes<PersonDocument>();
+        await this.AssertPersonDocumentIndexes<OtherDatabasePersonDocument>();
     }
 
-    private async Task AssertPersonDocumentIndexes()
+    private async Task AssertPersonDocumentIndexes<TDocument>()
     {
-        using var personDocumentIndexCursor = await this.Services.GetRequiredService<IMongoCollection<PersonDocument>>().Indexes.ListAsync();
+        using var personDocumentIndexCursor = await this.Services.GetRequiredService<IMongoCollection<TDocument>>().Indexes.ListAsync();
```

Comment:
can we assert that the resolved collection is indeed associated to the other database?

Reply:
Yes it will be checked in my next commit.